README.md

```diff
@@ -1 +1,137 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To-do-list App <a name="about-project">
+
+Hi there! Welcome to my To-do-list Web App</a>
+
+Stay organized and increase productivity with our intuitive to-do-list web app. Add, edit, and delete tasks with clear all functionality from any device.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [x] **[Add tasks]**
+- [x] **[List task]**
+- [x] **[Edit task name]**
+- [x] **[delete task]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://to-do-app-hddr.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install Node.js, Web browser, and code editor (recommended: Visual studio).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```snippet
+  git clone https://github.com/UbdaNam/react-todo-app.git
+```
+
+### Install
+
+```snippet
+  npm install
+```
+
+Install this project with: Any code editor which you are comfortable with
+
+### Usage
+
+In the project directory, you can run:
+
+```snippet
+  npm start
+```
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ubda Nam**
+
+- GitHub: [@UbdaNam](https://github.com/UbdaNam)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## Future Features <a name="future-features"></a>
+
+- [ ] **[Add navigation]**
+- [ ] **[Add reminder]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contribution <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/UbdaNam/react-todo-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project please give it a star. If you have any recommandations for future improvments, I would be happy to hear them.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
```

Comment:
[Optional] - removing unnecessary comments from a README file can make it more concise and easier to read, improving its effectiveness as a communication tool for the project. This can ultimately save time for both the author and the readers.